app/views/idv/shared/_document_capture.html.erb

```diff
@@ -57,6 +57,8 @@
       <li><%= t('doc_auth.tips.document_capture_id_text4') %></li>
     </ul>
 
+    <%= f.input :empty_input, as: :hidden %>
```

Comment:
I am still trying to figure out what to do here.

The doc auth base step `permit` method includes a `params.require(:doc_auth)` line. Previously, the data url inputs had empty values which created the `doc_auth` hash in the params. Now that is missing.

This breaks when the code to simulate an image upload API request tries to submit this form to advance past this page. I may end up changing that to work like the react component, and create it's own form and submit that.

Reply:
I tried dynamically creating a form and submitting that, but discovered a hiccup. It looks like the document capture component actually uses this form to submit the page. So, I added a hidden input here that submits the session UUID. We don't do anything with it, but it does add the expected params.